libs/cardano-ledger-core/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## 1.10.0.0
 
+* Add `estimateMinFeeTx`, `addDummyWitsTx` and move `setMinFeeTx` to `Cardano.Ledger.Tools`
```

Comment:
I wonder if we should mention the removal of these functions in the cardano-ledger-api changelog as well, since maybe someone using the api might not know to look into core to figure out what happened?

Reply:
`estimateMinFeeTx` is still re-exported in `cardano-ledger-api`.

Also, neither of those two functions are in any release on chaps yet.